src/components/PickerTypeDropdown.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from "prop-types";
+import Select from "react-select";
+import React from "react";
+
+export const PickerTypeDropdown = ({
```

Comment:
This probably needs a doc comment.

Also, are you sure we really need a new component for this? It looks like this is just a dropdown with two options that gets used once. Why isn't it the same component as e.g. the file type selection dropdown?